src/main/kotlin/no/fdk/dataset_catalog/rdf/Namespace.kt

```diff
@@ -102,4 +91,41 @@ class PROV {
         val qualifiedAttribution = model.createProperty("${uri}qualifiedAttribution")
 
     }
-}
\ No newline at end of file
+}
+
+class ExtendedXSD {
+    companion object {
+        private val model = ModelFactory.createDefaultModel()
+        val seeAlso = model.createProperty("${XSD.NS}seeAlso")
+    }
+}
+
+class ELI {
+    companion object {
+        const val uri = "http://data.europa.eu/eli/ontology#"
+
+        val LegalResource = ResourceFactory.createResource("${uri}LegalResource")
+    }
+}
+
+class CPSV {
+    companion object {
+        private val model = ModelFactory.createDefaultModel()
+        const val uri = "http://purl.org/vocab/cpsv#"
+
+        val Rule = ResourceFactory.createResource("${uri}Rule")
+
+        val follows = model.createProperty("${uri}follows")
+        val implements = model.createProperty("${uri}implements")
```

Comment:
Kan bruke ResourceFactory direkte her i stedet for model

Reply:
0d499fd